WGS_pipeline/cnn-variant-common-tasks.wdl

```diff
@@ -0,0 +1,358 @@
+workflow cnn_variant_common_tasks {}
```

Comment:
@jkobject  did this eventually work? It failed again on my end:
https://app.terra.bio/#workspaces/broad-firecloud-ccle/DepMap_WGS_CN_tmp/job_history/ec5526e0-a3ee-4d35-bd99-afeb742bd65b